### DISCUSSION TITLE
Hosted Version Documentation

### DISCUSSION THREAD

**Discussion by jcantrill** [vectordotdev/vector#19665] [Q&A] [answered]
Is there a site that hosts the documentation by released version?

**Comment by jszwedko** [comment, marked as answer]
> Is there a site that hosts the documentation by released version?

Unfortunately not. We used to have this with our old site hosting, but lost it transition to the new hosting and haven't been able to re-add it yet. On the plus side, the documentation is versioned with the code, so you should be able to check the git tag you are interested and run `make serve` from the the `websites` directory. There are a few prerequisites mentioned here: https://github.com/vectordotdev/vector/tree/master/website#prerequisites.
